docs/learn-account-generation.md

```diff
@@ -348,3 +349,28 @@ computer a very small and limited scope for interacting with the Parity Signer d
 continue to keep your key safe.
 
 ![Parity Signer Create Account 6](assets/accounts/ps-create-6.jpg)
+
+## Vanity Generator
+
+Vanity generator is a tool on [Polkadot-JS UI](https://polkadot.js.org/apps/#/accounts/vanity) that
+lets you generate addresses which contain some specific substring.
+
+![Vanity Generator page](assets/polkadot_vanity_generator_page.png)
+
+Components of vanity generator:
+
+- `Search for`: Some substring that you would like to include in your new address.
+- `case sensitive`: Yes or no
+- `keypair crypto type`: Specifies the type of account you'd like to generate, Schnorrkel is
```

Comment:
```suggestion
- "keypair crypto type": Specifies the type of account you'd like to generate, Schnorrkel is
```

Reply:
I think we should reserve backticks for specifically for CLI commands and code snippets. Here it is describing the title of inputs so quotations are more proper. NIT